csi/rockcraft.yaml

```
@@ -0,0 +1,60 @@
+# Dockerfile: https://github.com/kubeflow/model-registry/blob/v0.2.22/cmd/csi/Dockerfile.csi
+
+name: model-registry-csi
```

suggestion(non-blocking): 
```suggestion
name: model-registry-storage-initializer
```
just for clarity and to be easy to map to upstream (they call it `model-registry-storage-initializer`)